app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class UserSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :id, :email
+end
```

Comment:
Would we want any of the user relationships to be included here?

Reply:
Good question! I didn't include it yet as was getting basics sent. We can certainly add it that information if that is the direction we want to go.

Reply:
I'm okay with keeping it with the basics until we need more!